src/__test__/views/NewLanding.test.tsx

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-undef */
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import { render } from '@testing-library/react';
+import renderer from 'react-test-renderer';
+import NewLanding from '../../views/NewLanding';
+import { AuthProvider } from '../../contexts/AuthContext';
+
+describe('NewLanding component test', () => {
+  it('should have the correct title', () => {
+    const { getByTestId } = render(
```

Comment:
Searching about the `getByTestId` gave many results talking about how often it's used in detriment of better, more meaningful queries.

For instance, given that accessibility an important aspect of our application, we could use the [getByRole](https://testing-library.com/docs/queries/byrole/) query instead: we look for what our user would be looking for (a button, a heading, etc. — or what they would be "told about"; maybe we could use some help from the design team here), and see if it contains what it should contain.

Reply:
Great point. I think we can discuss this item.